client/src/components/OptionsDrawer/OptionsDrawer.tsx

```diff
@@ -28,7 +30,7 @@ const OptionsDrawer = ({ open, setOpen }: Props): JSX.Element => {
             key={board._id}
             button
             onClick={() => {
-              fetchBoard(board._id);
+              history.push(`/dashboard/board/${board._id}`);
               setOpen(false);
```

Comment:
Would `setOpen` still be necessary if you load a new board completely from the `history.push`?

Reply:
Probably not, thanks!

Reply:
Just tested and it's needed 😮